extension/content.js

```diff
@@ -344,6 +344,18 @@ function addFilterCommentsByYou() {
 	}
 }
 
+function addProjectNewLink() {
+	const projectNewLink = `<a href="/${repoUrl}/projects/new" class="btn btn-sm" id="refined-github-project-new-link">Add a project</a>`;
+	$('#projects-feature').next('p.note').after(projectNewLink);
```

Comment:
`$('#projects-feature').siblings('p.note').after(projectNewLink);`

`.next()` only works on immediately following siblings.

Also, we need to check if the button was added and not add it again.

![screen shot 2017-05-09 at 1 17 19 pm](https://cloud.githubusercontent.com/assets/737065/25863437/e13fe5b8-34b9-11e7-937f-99aef57c94fb.png)